llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -257,6 +257,146 @@ def simm12 : RISCVSImmLeafOp<12> {
   }];
 }
 
+// A 7-bit unsigned immediate where the least significant two bits are zero.
```

Comment:
Why do these need to move? RISCVInstrInfoXMips.td is included after RISCVInstrInfoC.td